lib/ember_cli/shell.rb

```diff
@@ -50,6 +52,23 @@ def test
     attr_accessor :pid
     attr_reader :ember, :env, :options, :paths
 
+    def invalid_ember_dependencies?
+      !run("#{paths.ember} version")
+    rescue DependencyError
```

Comment:
This may need a comment or something else to explain the intent. This is to cover the edge case of the very first installation, before npm has installed ember itself for the first time.